readme.md

```diff
@@ -140,6 +140,7 @@ Each of the generator properties (like `name`, `address`, and `lorem`) are calle
     firstNameMale                             // 'Maynard'
     firstNameFemale                           // 'Rachel'
     lastName                                  // 'Zulauf'
+    ssn                                       // '123-45-6789'
```

Comment:
Apologies, this section is labeled with a locale-specific provider already (`Faker\Provider\en_US\Person`), so I assumed it was the right place to put it. Should I create a new `Faker\Provider\en_US\Person` group in the general section for this?